torchserve_sanity.sh

```diff
@@ -56,8 +56,8 @@ start_torchserve
 models=("fastrcnn" "fcn_resnet_101" "my_text_classifier_v2" "resnet-18" "my_text_classifier_scripted_v2" "alexnet_scripted" "fcn_resnet_101_scripted"
```

Comment:
Should we include here the `deeplabv3_resnet_101` and `deeplabv3_resnet_101_scripted` models so that the sanity checks also run over the DeepLabV3 model?

Reply:
This may not be required as not all the models are included on the sanity checks, but it should be a nice feature to have, even though the build will fail unless the model MAR is included in the https://torchserve.pytorch.org/mar_files host.

Reply:
The model has already been uploaded to a Dropbox shared file via URL, so that anyone can download it from the following URL: https://www.dropbox.com/s/2otj6ssf69bnb33/deeplabv3_resnet_101.mar?dl=1

You can also check it on the [docs/model_zoo.md](https://github.com/alvarobartt/serve/blob/deeplab-example/docs/model_zoo.md) updated table.

Reply:
Postponed for the next iteration.